snippets/distr-tracing-tempo-tempostack-custom-resource.adoc

```diff
@@ -12,41 +12,42 @@ kind: TempoStack # <1>
 metadata:
   name: simplest
   namespace: <project_of_tempostack_instance> # <2>
-spec:
-  storage: # <3>
-    secret: # <4>
-      name: <secret_name> # <5>
-      type: <secret_provider> # <6>
-  storageSize: <value>Gi # <7>
+spec: <3>
+  storage: # <4>
+    secret: # <5>
+      name: <secret_name> # <6>
+      type: <secret_provider> # <7>
+  storageSize: <value>Gi # <8>
   resources:
     total:
       limits:
         memory: 2Gi
         cpu: 2000m
   tenants:
-    mode: openshift # <8>
-    authentication: # <9>
-      - tenantName: dev # <10>
-        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfa" # <11>
+    mode: openshift # <9>
+    authentication: # <10>
+      - tenantName: dev # <11>
+        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfa" # <12>
       - tenantName: prod
         tenantId: "1610b0c3-c509-4592-a256-a1871353dbfb"
   template:
     gateway:
-      enabled: true # <12>
+      enabled: true # <13>
     queryFrontend:
       jaegerQuery:
-        enabled: true # <13>
+        enabled: true # <14>
 ----
 <1> This CR creates a TempoStack deployment, which is configured to receive Jaeger Thrift over the HTTP and OpenTelemetry Protocol (OTLP).
 <2> The namespace that you have chosen for the TempoStack deployment.
-<3> Specifies the storage for storing traces.
-<4> The secret you created in step 2 for the object storage that had been set up as one of the prerequisites.
-<5> The value of the `name` field in the `metadata` section of the secret. For example: `minio`.
-<6> The accepted values are `azure` for Azure Blob Storage; `gcs` for Google Cloud Storage; and `s3` for Amazon S3, MinIO, or {odf-full}. For example: `s3`.
-<7> The size of the persistent volume claim for the Tempo Write-Ahead Logging (WAL). The default is `10Gi`. For example: `1Gi`.
-<8> The value must be `openshift`.
-<9> The list of tenants.
-<10> The tenant name, which is to be provided in the `X-Scope-OrgId` header when ingesting the data.
-<11> The unique identifier of the tenant. Must be unique throughout the lifecycle of the TempoStack deployment. The {TempoShortName} uses this ID to prefix objects in the object storage. You can reuse the value of the UUID or `tempoName` field.
-<12> Enables a gateway that performs authentication and authorization. The Jaeger UI is exposed at `+http://<gateway_ingress>/api/traces/v1/<tenant_name>/search+`.
-<13> Exposes the Jaeger UI, which visualizes the data, via a route.
+<3> Red{nbsp}Hat supports only the custom resource options that are available in the {DTProductName} documentation.
```

Comment:
This line and a similar line in the other file is the only content added. Git shows the rest of the changes as new content only due to the callout renumbering.